terraform/modules/emr-launcher/lambda_monitoring.tf

```diff
@@ -0,0 +1,24 @@
+resource "aws_cloudwatch_metric_alarm" "uc_export_to_crown_controller_lambda_failure" {
```

Comment:
Why is this resource called uc_export_to_crown_controller_lambda_failure?

Reply:
because I am an idiot